### DISCUSSION TITLE
Json to Single string

### DISCUSSION THREAD

**Discussion by OwainWilliams** [DevToys-app/DevToys#631] [Ideas]
Hi, 
I was just wondering if there was a way to do this already within DevToys. 

Take a load of json and then convert it to a single string, removing all spaces and line breaks e.g. 
https://jsontostring.com/convert-json-to-one-line/

I have a need for this on a regular basis and then I use DevToys to escape this string. To be able to do it all in one app would be super handy.

**Comment by jamescurran** [comment]
In the JSon Formatter tool, in the Indentation option, has a choice for "Minified".  Does that do what you want?

**Reply by OwainWilliams** [reply]
I was about to test it but looks like there is a bug in the latest version :) #620 

I think I tried it but didn't give the result I was looking for. I'll test again once this bug is fixed.
